pandas/tests/frame/test_convert_to.py

```diff
@@ -190,17 +147,65 @@ def test_to_records_with_unicode_column_names(self):
         )
         tm.assert_almost_equal(result, expected)
 
+    @pytest.mark.parametrize('mapping', [
+        dict,
+        collections.defaultdict(list),
+        collections.OrderedDict])
+    def test_to_dict(self, mapping):
```

Comment:
Done - I added one at the end.  Let me know if that is what you were getting at.